libs/community/langchain_community/chat_models/openai.py

```diff
@@ -335,12 +325,20 @@ def validate_environment(cls, values: Dict) -> Dict:
     @property
     def _default_params(self) -> Dict[str, Any]:
         """Get the default parameters for calling OpenAI API."""
+        if is_openai_v1():
+            extra_params = {
+                "extra_query": self.extra_query,
+                "extra_headers": self.extra_headers,
+                "extra_body": self.extra_body,
+            }
+        else:
+            extra_params = self.model_kwargs
         params = {
             "model": self.model_name,
             "stream": self.streaming,
             "n": self.n,
             "temperature": self.temperature,
-            **self.model_kwargs,
+            **extra_params,
```

Comment:
I think this might be a breaking change unless I'm misunderstanding something. Why can't you just pass in `model_kwargs={"extra_body": {...}}` for what you need this for?